package.json

```diff
@@ -18,28 +18,50 @@
     "scripts": {
         "dev": "rollup -m -c",
         "build": "rimraf build && rollup -m -c --environment NODE_ENV:production",
-        "test": "mocha"
+        "postbuild": "echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
+        "test": "npm run test:node && npm run test:browser",
+        "test:node": "NODE_OPTIONS='--experimental-vm-modules --no-warnings' jest --coverage",
+        "test:browser": "NODE_OPTIONS='--experimental-vm-modules --no-warnings' jest --coverage --testEnvironment=jsdom"
     },
     "author": "Jose F. Romaniello <jfromaniello@gmail.com> (http://joseoncode.com)",
     "contributors": [
         "Sam Bellen <sam.bellen@auth0.com>"
     ],
     "license": "MIT",
     "devDependencies": {
+        "@babel/core": "^7.22.7",
+        "@babel/preset-env": "^7.22.7",
         "@rollup/plugin-commonjs": "^21.0.2",
         "@rollup/plugin-node-resolve": "^13.1.3",
-        "expect.js": "^0.3.1",
-        "mocha": "^9.2.1",
+        "@types/jest": "^29.5.3",
+        "babel-jest": "^29.6.0",
+        "jest": "^29.6.0",
```

Comment:
Does Jest need to be brought in for this? Perhaps the [Node.js test runner](https://nodejs.org/api/test.html) would suffice here?

Reply:
We use jest in all our SDKs, so we bring in jest for aligning with others.